projects/configs/tag-operations/mr-spectroscopy.yaml

```diff
@@ -0,0 +1,29 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Specific tag operations for spectroscopy data projects
+
+#################################### 0018 Group ###################################
+#
+#
+- name: "RF Echo Train Length Attribute"
+  group: 0x0018
+  element: 0x9240
+  op: "keep"
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189112/00189240
+- name: "MR Timing and Related Parameters Sequence Attribute"
+  group: 0x0018
+  element: 0x9112
+  op: "keep"
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189112
```

Comment:
Are these the only MR spectroscopy tags that they would need? Seems surprising that they'd only need two extra data points

Reply:
Consider these commits as a starting point: [1be4168](https://github.com/SAFEHR-data/PIXL/pull/513/commits/1be41683aae104a9eb4a268e3c3cc9c1cd9aa771)